Makefile

```diff
@@ -44,6 +44,8 @@ servewallet-regtest:
 	go install ./cmd/servewallet/... && servewallet -regtest
 servewallet-multisig:
 	go install ./cmd/servewallet/... && servewallet -multisig
+servewallet-prod:
+	go install ./cmd/servewallet/... && servewallet -devserver=false
```

Comment:
`-devservers`, as you use the plural everywhere else too

Reply:
Fixed.